install/kubernetes/Makefile.values

```diff
@@ -37,8 +37,8 @@ export CILIUM_NODEINIT_DIGEST:=sha256:8d7b41c4ca45860254b3c19e20210462ef89479bb6
 
 # renovate: datasource=docker
 export CILIUM_ENVOY_REPO:=quay.io/cilium/cilium-envoy
-export CILIUM_ENVOY_VERSION:=v1.31.4-1734096493-fff09f16c2c269b22509c86dfc1d3e8f52eb3857
-export CILIUM_ENVOY_DIGEST:=sha256:4ca99b3a1c18b1e5c498e1d1fc55b0f90e2914567ffdb12bedd9b32970501c08
+export CILIUM_ENVOY_VERSION:=v1.31.4-3a8ccd54545785689e677d785b69025d1d4b33de
```

Comment:
can you use the one with timestamp tag, so that renovate can perform automatic update later?

Reply:
updated, thanks (update script doesn't respect this :cry: will look into this)